spec/pg/result_spec.rb

```diff
@@ -307,6 +307,28 @@
 		expect( sqlstate ).to eq( 22012 )
 	end
 
+	it "provides the error message" do
+		@conn.send_query("SELECT xyz")
+		res = @conn.get_result; @conn.get_result
+		expect( res.error_message ).to match(/"xyz"/)
+		expect( res.result_error_message ).to match(/"xyz"/)
+	end
+
+	it "provides a verbose error message", :postgresql_96 do
+		@conn.send_query("SELECT xyz")
+		res = @conn.get_result; @conn.get_result
+		# PQERRORS_TERSE should give a single line result
+		expect( res.verbose_error_message(PG::PQERRORS_TERSE, PG::PQSHOW_CONTEXT_ALWAYS) ).to match(/\A.*\n\z/)
```

Comment:
It's not a blocker but I think to assert that there's only one newline (at the end) would look like `/\A[^\n]*\n\z/`.

Reply:
The dot in a Regex without postfix "m" essentially means everything except "\n". So `/\A.*\n\z/` should be the same as `/\A[^\n]*\n\z/m` , I think.